docs/_includes/js/affix.html

```diff
@@ -2,7 +2,7 @@
   <h1 id="affix" class="page-header">Affix <small>affix.js</small></h1>
 
   <h2 id="affix-examples">Example</h2>
-  <p>The subnavigation on the right is a live demo of the affix plugin.</p>
+  <p>The affix plugin allows you to easily create a fixed positioned element. The subnavigation on the right is a live demo of the affix plugin.</p>
```

Comment:
This doesn't seem quite accurate. The important thing is that it toggles `position: fixed` on and off, so as to emulate something like `position: sticky;` (except it was created before the advent of `position: sticky`).